cx-server-companion/files/cx-server-companion.sh

```diff
@@ -590,6 +591,10 @@ function start_jenkins_container()
         if [ -e /cx-server/mount/jenkins-configuration ]; then
             environment_variable_parameters+=(-e CASC_JENKINS_CONFIG=/var/cx-server/jenkins-configuration)
         fi
+
+        environment_variable_parameters+=(-e PPIPER_INFRA_IT_CF_PASSWORD)
+        environment_variable_parameters+=(-e PPIPER_INFRA_IT_CF_USERNAME)
```

Comment:
Is the cx-server-companion part of the infrastructure tests or also something productive?
I wonder why parameters from the tests are mentioned in the script.

Reply:
This is the productive life cycle mgmt script, but I don't see how we can get by without passing the environment for testing. Also, I don't see a big issue there. We need a way to instrument our environment for testing.

Reply:
Yes it is productive code. But here you can argue that it is not show to the user but hidden inside the docker container as a private API.

Reply:
Ok, thanks for clarification